homeassistant/components/matter/light.py

```diff
@@ -295,7 +295,11 @@ def _update_from_device(self) -> None:
             # brightness support
             if self._entity_info.endpoint.has_attribute(
                 None, clusters.LevelControl.Attributes.CurrentLevel
-            ):
+            ) and self._entity_info.endpoint.device_types != {device_types.OnOffLight}:
+                # if endpoint has LevelControl and is not OnOffLight device type
+                # this filter is important because OnOffLight can have an optional LevelControl
```

Comment:
I don't understand this comment. It's talking about devices that are not OnOffLight, but then says that the filter is important for OnOffLight.

Reply:
Please limit the comment lines to max 88 characters per line.

Reply:
fixed the comment